packages/experimental/src/components/BreadcrumbWithOverflow/BreadcrumbWithOverflow.stories.js

```diff
@@ -0,0 +1,72 @@
+//
```

Comment:
Clicking on a breadcrumb reloads the storybook. Maybe it can show an event on the actions tab instead?